src/pages/who-we-are/_DerivNumbers.tsx

```diff
@@ -125,7 +125,7 @@ const DerivNumbers = () => {
                 >
                     {deriv_numbers.map((number) =>
                         number.map(({ count, title }) => (
-                            <Flex key={count} fd="column" height="unset">
+                            <Flex key={title.props.translate_text} fd="column" height="unset">
```

Comment:
use optional chaining so that you would not face unwanted errors.

Reply:
@hirad-rewok these data are static, I think we don't need optional chaining here